### ISSUE TITLE
Problem with import xml zabbix-3.0

### ISSUE THREAD

**Issue by kirush0280** [aGGre55or/zabbix-app-bind#1]

**Comment by aGGre55or** [comment]
Describe your problem and if you show a screenshot.

**Comment by j040n3t0** [comment]
Hi,

To use this template I need install Zabbix and Bind in the same server?

Thanks.

**Comment by aGGre55or** [comment]
No. The script (named-stats.sh) are executed from zabbix-agent installed on a server with Bind. 
Zabbix-server refers to zabbix-agent through IP:PORT and takes the values for visualization.
